zproject/settings.py

```diff
@@ -158,6 +158,7 @@ def get_secret(key):
                     'ZULIP_COM': False,
                     'ZULIP_COM_STAGING': False,
                     'STATSD_HOST': '',
+                    'OPEN_REALM_CREATION': False,
```

Comment:
You should also set `OPEN_REALM_CREATION=True` in the development settings near the `from local_settings import *` line higher up in this file, so the setting defaults to on for development